odoo/addons/base/models/ir_cron.py

```diff
@@ -64,11 +70,11 @@ class ir_cron(models.Model):
                                       ('days', 'Days'),
                                       ('weeks', 'Weeks'),
                                       ('months', 'Months')], string='Interval Unit', default='months')
-    numbercall = fields.Integer(string='Number of Calls', default=1, help='How many times the method is called,\na negative number indicates no limit.')
-    doall = fields.Boolean(string='Repeat Missed', help="Specify if missed occurrences should be executed when the server restarts.")
     nextcall = fields.Datetime(string='Next Execution Date', required=True, default=fields.Datetime.now, help="Next planned execution date for this job.")
     lastcall = fields.Datetime(string='Last Execution Date', help="Previous time the cron ran successfully, provided to the job through the context on the `lastcall` key")
     priority = fields.Integer(default=5, help='The priority of the job, as an integer: 0 means higher priority, 10 means lower priority.')
+    failures = fields.Integer(default=0, help='The number of consecutive failures of this job')
+    first_failure_date = fields.Datetime(string='First Failed Date', help='The first time the cron failed')
```

Comment:
First *failure* date or first *failed* date? Please make it consistent. Also reading the field name and the help message my intuition tells me it is the "first failure date *ever*" not the "consecutive failures starting date".
```suggestion
    consecutive_failures_starting_date = fields.Datetime(string="Failure date", help="The moment this job started to fail. It resets everytime the job succeeds.")
```